nuxt/components/testimonial-grid.vue

```diff
@@ -1,23 +1,19 @@
 <template>
-  <section :class="containerClasses"
-    class="testimonial-list col-lg-8 row-mt-4 mx-auto justify-content-center container">
-    <component :is="'h' + headlineLevel" v-if="headline" class="testimonial-list__headline space-bottom-1">
+  <section :class="containerClasses" class="col-lg-8 row-mt-4 mx-auto justify-content-center container">
+    <component :is="'h' + headlineLevel" v-if="headline" class="space-bottom-1">
```

Comment:
You can just use the headline component here and provide the level to it so it will pick the right tag. This way you will just render not components but just basic h1 tags without the headline functionalities.